gradle.properties

```diff
@@ -19,7 +19,10 @@ commonsLangVersion=3.5
 snakeyamlVersion=1.29
 fuelVersion=2.3.1
 
-jacksonVersion = 2.12.5
+# jacksonDatabindVersion can be collapsed back into Jackson Version once 2.13.3 is out
+jacksonDatabindVersion = 2.13.2.2
+jacksonVersion = 2.13.2
```

Comment:
2.13.3 has been out for a while (https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind).

It also doesn't look like you reference the databind lib?